cmd/k6config.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+// structure of the config file with the fields that are used by k6exec
+type k6configFile struct {
+	Collectors struct {
+		Cloud struct {
+			Token string `json:"token"`
+		} `json:"cloud"`
+	} `json:"collectors"`
+}
+
+// loadConfig loads the k6 config file from the given path or the default location.
+// if using the default location and the file does not exist, it returns an empty config.
+func loadConfig(configPath string) (k6configFile, error) {
+	var (
+		config       k6configFile
+		usingDefault bool
+		homeDir      string
+		err          error
+	)
+
+	if configPath == "" {
+		usingDefault = true
+		homeDir, err = os.UserConfigDir() //nolint:forbidigo
+		if err != nil {
+			return config, fmt.Errorf("failed to get user home directory: %w", err)
+		}
+		configPath = filepath.Join(homeDir, "loadimpact", "k6", "config.json")
```

Comment:
Like I said in DM, but it's likely possible that soonish we have to support a new path `~/.config/k6/...` which should have a precedence

Reply:
Maybe we can fo this in a follow up PR?